api/v1/policy_webhook.go

```diff
@@ -34,10 +35,10 @@ func (r *Policy) SetupWebhookWithManager(mgr ctrl.Manager) error {
 
 //+kubebuilder:webhook:path=/validate-policy-open-cluster-management-io-v1-policy,mutating=false,failurePolicy=Ignore,sideEffects=None,groups=policy.open-cluster-management.io,resources=policies,verbs=create,versions=v1,name=policy.open-cluster-management.io.webhook,admissionReviewVersions=v1
 
-var _ webhook.Validator = &Policy{}
+var _ webhook.CustomValidator = &Policy{}
 
-// ValidateCreate implements webhook.Validator so a webhook will be registered for the type
-func (r *Policy) ValidateCreate() (admission.Warnings, error) {
+// ValidateCreate implements webhook.CustomValidator so a webhook will be registered for the type
+func (r *Policy) ValidateCreate(ctx context.Context, obj runtime.Object) (admission.Warnings, error) {
```

Comment:
```suggestion
func (r *Policy) ValidateCreate(ctx context.Context, _ runtime.Object) (admission.Warnings, error) {
```
since obj is not used

Reply:
Good call. Probably the same for the `context`?